packages/cozy-pouch-link/src/PouchManager.js

```diff
@@ -111,12 +111,9 @@ class PouchManager {
     this.removeListeners()
     await this.clearSyncedDoctypes()
     await this.clearWarmedUpQueries()
-    await this.storage.destroyAllDoctypeLastSequence()
-    await this.storage.destroyAllLastReplicatedDocID()
+    await this.storage.destroy()
 
-    return Promise.all(
-      Object.values(this.pouches).map(pouch => pouch.destroy())
-    )
+    await allSettled(Object.values(this.pouches).map(pouch => pouch.destroy()))
```

Comment:
There's nothing left to return. Is this intentional ?

Reply:
Not really, I changed it